### ISSUE TITLE
Split arguments from command

### ISSUE THREAD

**Issue by piranna** [arve0/system-install#1]
Instead of a string, return an object (maybe with a `.toString()` function?) that has separately the command from its arguments, and ideally a `needSudo` attribute to know if its needed. This would allow more flexibility on its usage, specially regarding to its execution later with `child_process.spawn()`. I would be able to do this and other optimization and send a pull-request if you want.

**Comment by arve0** [comment]
This is a good idea, and I had this in mind, but went for the simplest approach.

I would prefer if the API stayed the same, and add extra features as additional methods / properties. In example:

``` js
const mngr = require('system-install');

mngr();  // 'sudo apt-get install'
mngr.needSudo();  // true
mngt.install('pandoc');  // 'sudo apt-get install pandoc'
```

**Comment by piranna** [comment]
Yeah, it makes sense... When requiring the module it would return a function as it does at this moment, but having some read-only properties (instead of getters) to define the environment like sudo, command... since they will never change for a single machine (except if they install a new package manager, but that's a very extrange corner case). How do you see it?

**Comment by arve0** [comment]
:+1:
